sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/CameraInfo camera_info
geometry_msgs/TransformStamped pose
